cdap-app-fabric/src/test/java/co/cask/cdap/app/runtime/monitor/RuntimeMonitorTest.java

```diff
@@ -72,33 +82,34 @@
 
   private static CConfiguration cConf;
   private static MessagingService messagingService;
+
+  private final AtomicReference<ProgramRunId> publishProgramKilled = new AtomicReference<>();
+
   private RuntimeMonitorServer runtimeServer;
   private MultiThreadMessagingContext messagingContext;
   private TransactionManager txManager;
-  private TransactionSystemClient transactionSystemClient;
   private DatasetService datasetService;
   private DatasetFramework datasetFramework;
   private Transactional transactional;
 
   private KeyStore serverKeyStore;
   private KeyStore clientKeyStore;
 
+
   @ClassRule
   public static final TemporaryFolder TMP_FOLDER = new TemporaryFolder();
 
   @Before
   public void init() throws IOException, TopicAlreadyExistsException {
     cConf = CConfiguration.create();
     cConf.set(Constants.CFG_LOCAL_DATA_DIR, TMP_FOLDER.newFolder().getAbsolutePath());
-    cConf.set(Constants.AppFabric.PROGRAM_STATUS_EVENT_TOPIC, "programStatus");
```

Comment:
Why are we removing this? It was used so that `RuntimeMonitor` reads from a different TMS topic and writes to different TMS topic with single TMS instance.

Reply:
We only need to change one. The cConf already has one set by default. For testing purpose, we only need to set one that used by the runtime monitor for republishing

Reply:
I see.